main.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224510842-be753586-5623-4b23-a5ec-5a5dccc4e692.png)

**[OPTIONAL]**

- [ ] It would be nice if you make all your commit messages Capitalize to make your pull request descriptive and professional 😊